### PR TITLE
Updates links in the source, updates readme and modifies testimages function

### PR DESCRIPTION
**Pull request by mronian**
Changes in this PR -
- Modifies links in build.jl
- Now Pkg.build("TestImages") will add only the standard image dataset, all other images can be added by calling the testimage function with the image name as required.
- testimage function checks both local directory and repository for an image before giving an error
- Updates Readme

### CODE REVIEW

**Comment by mronian** [comment]
@timholy I am not sure, should I be merging my own pull requests?

File: src/TestImages.jl [hunk 1]
```diff
@@ -17,9 +56,22 @@ function testimage(filename, ops...)
                 break
             end
         end
-        havefile || error("$filename not found. Here are the contents of the images/ directory:\n$(join(fls, '\n'))")
+
+        if !havefile
+            info("Could not file in directory. Checking if it exists in the online repository.")
```
Comment (timholy) [inline src/TestImages.jl]: "Could not *find* file..."?

**Comment by timholy** [comment]
Looks very nice.

In general, it's probably not a bad idea for a 2nd pair of eyes to look it over. Most of us who have been working on julia for a while, and have commit access to the main repo or one or more packages, still tend to prefer to get at least one review. That said, if I you need something merged ASAP (because, e.g., something else you want to change depends on it) or if I'm just plain slow, feel free to merge. Or if others make PRs and you want to review them and make the decision about merging, by all means do so.

File: src/TestImages.jl (hunk 1 above)
Comment (mronian) [inline src/TestImages.jl]: Corrected.

**Comment by mronian** [comment]
I'll keep that in mind. Since this PR has been reviewed, I guess this gives me a chance to do my first merge. I'll wait for a few hours before merging in case anyone wants to add comments.

**Pull request merged**